src/App.tsx

```diff
@@ -40,6 +40,6 @@ const appRouter = createBrowserRouter([
 	},
 ]);
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-
-root.render(<RouterProvider router={appRouter} />);
+const element: HTMLElement | null = document.getElementById('root');
```

Comment:
what is going on here?

Reply:
`element` is of type `HTMLElement` or `null` and assigned the value of `document.getElementById("root")`.